pyodide-build/pyodide_build/doit_tasks.py

```diff
@@ -0,0 +1,426 @@
+import argparse
+import os
+import shutil
+from pathlib import Path
+
+from pyodide_build import buildall, buildpkg
+from pyodide_build.buildpkg import get_bash_runner
+from pyodide_build.common import get_make_flag, search_pyodide_root
+
+# from doit.reporter import ZeroReporter
+
+DOIT_CONFIG = {
+    "verbosity": 2,
+    "minversion": "0.36.0",
+    "action_string_formatting": "both",
+    # "reporter": ZeroReporter,
```

Comment:
A other subjective, but hard-won config insights:
- `"par_type": "thread",`
  - this is the only one that works correctly across all platforms
- `"backend": "sqlite3",
  - this is the least likely to get corrupted
  - it only creates one file